spec.bs

```diff
@@ -937,6 +986,30 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
       1. Set |auctionConfig|'s [=auction config/seller signals=] to the result of
         [=serializing a JavaScript value to a JSON string=], given |result|.
     * To handle an error, set |auctionConfig|'s [=auction config/seller signals=] to failure.
+1. If |config|["{{AuctionAdConfig/auctionNonce}}"] [=map/exists=]:
+  1. Let |nonce| be an empty [=string=].
+  1. [=list/For each=] |i| in [=the range=] from 0 to the [=string/length=] of
+    |config|["{{AuctionAdConfig/auctionNonce}}"], exclusive:
+    1. Let |c| be the |i|th [=code unit=] of |config|["{{AuctionAdConfig/auctionNonce}}"] if it is
+      0x0000 NULL or in the range 0x0020 SPACE to 0x007F DELETE, inclusive, otherwise "`?`".
+    1. Append |c| to the end of |nonce|.
+  1. Set |auctionConfig|'s [=auction config/auction nonce=] to the result of running
+    [=get uuid from string=] with |nonce|.
+1. If |config|["{{AuctionAdConfig/additionalBids}}"] [=map/exists=]:
+  1. [=exception/Throw=] a {{TypeError}} if any of the following conditions hold:
+    * |config|["{{AuctionAdConfig/auctionNonce}}"] does not [=map/exist=];
+    * |config|["{{AuctionAdConfig/interestGroupBuyers}}"] does not [=map/exist=], or it's [=list/empty=];
+    * |config|["{{AuctionAdConfig/componentAuctions}}"] is not [=list/empty=].
+  1. [=Handle an input promise in configuration=] given |auctionConfig| and
+    |config|["{{AuctionAdConfig/additionalBids}}"]:
+    * To parse the value |result|:
+      1. Set |auctionConfig|'s [=auction config/expects additional bids=] to true.
+      1. Set |auctionConfig|'s [=auction config/config idl=]["{{AuctionAdConfig/additionalBids}}"]
+        to {{undefined}}.

```

Comment:
In terms of what the seller worklet sees, but maybe I am making incorrect assumptions on how config idl is used?

Reply:
The [=auction config/config idl=] is what's passed to [evaluating a scoring script](https://github.com/WICG/turtledove/blob/main/spec.bs#L1479).

Removed the line that sets it to undefined when handling error, since waiting promise method will fail before the  |auctionConfig|'s [=auction config/config idl=]["{{AuctionAdConfig/additionalBids}}"] is used.